.circleci/config.yml

```diff
@@ -18,12 +18,19 @@ jobs:
           name: Install dependencies
           command: |
               npm config set strict-ssl false
+              NODE_VERSION=$(node --version)
+              if [[ ${NODE_VERSION:0:3} != "v6." ]]; then
+                # Avoid downloading chromium unnecessarily
+                npm config set ignore-scripts true
+              fi
               npm install
+              npm config set ignore-scripts false
       - run:
           name: Pre-Test
-          # ESLint only supports Node >=4
+          # ESLint only supports Node >=4 and Mochify requires Node >= 6
           command: |
-              if [[ ! "$(node --version)" =~ ^v4 ]]; then
+              NODE_VERSION=$(node --version)
+              if [[ ${NODE_VERSION:0:3} == "v6." ]]; then
```

Comment:
MINOR: `if [[ "$(node --version)" =~ ^v6 ]]; then`

Reply:
Yeah, that is what I tried first, but it failed saying that `=~` was an invalid operator.

Reply:
Oh, fair enough. I didn't look at the history.